samples/ChatRoom/ChatRoom.Common/ChatMsg.cs

```diff
@@ -1,9 +1,13 @@
 namespace ChatRoom;
 
-[Serializable]
-public record class ChatMsg(string? Author, string Text)
+[GenerateSerializer]
+public record class ChatMsg(
+    string? Author,
+    string Text)
 {
+    [Id(2)]
```

Comment:
Worth a comment to say something like
```suggestion
    [Id(2)]    // for records the positional constructor parameters get the Id by default
```

At least something that makes it obvious why it starts here with `2`.

Reply:
Updated this to be `0,` and `1`.